android/src/main/java/mediabrowser/apiinteraction/connectionmanager/ConnectionManager.java

```diff
@@ -751,39 +751,11 @@ public void ExchangePin(PinCreationResult pin, final Response<PinExchangeResult>
     }
 
     public void GetRegistrationInfo(final String featureName, String serverId, String localUsername, final Response<RegistrationInfo> response) {
```

Comment:
Shouldn't it just be removed? The API method has already been nulled on the server side

Reply:
Yes, but that could introduce errors if the apps are using this method. I have added an annotation to indicate the method is deprecated.